src/game.cpp

```diff
@@ -28,13 +29,24 @@ enum GameStatusFlag {
   FLAG_ENDLESS_MODE,
   FLAG_GAME_IS_ASKING_QUESTION_MODE,
   FLAG_QUESTION_STAY_OR_QUIT,
+  FLAG_TILES_REMOVED,           // Indicates if tiles have already been removed
```

Comment:
Pedantic, but good for uniformity: let's remove the multiple spaces before the comment here. Best to move the comment above `FLAG_TILES_REMOVED`.